core/src/main/resources/hudson/Messages_pt_BR.properties

```diff
@@ -76,3 +76,38 @@ ProxyConfiguration.FailedToConnect=Falha ao conectar em {0} (c\u00f3digo {1}).
 ProxyConfiguration.Success=Sucesso
 # Did not manage to validate {0} (may be too slow)
 FilePath.did_not_manage_to_validate_may_be_too_sl=N\u00e3o conseguiu validar {0} (pode ser muito lento)
+# {0} v{1} is missing. To fix, install v{1} or later.
+PluginWrapper.missing={0} v{1} est\u00E1 faltando. Para corrigir, instale v{1} ou superior.
+# No Exception details
+Functions.NoExceptionDetails=Sem detalhes da exce\u00E7\u00E3o
+# \ 
+# Whitespace can no longer be used as the separator. Please Use \u2018,\u2019 as the separator instead.
+FilePath.validateAntFileMask.whitespaceSeparator=Espa\u00E7o em branco n\u00E3o podem mais serrem utilizados como separador. Por favor use  \u2018, \u2019 como separadores.
+# Ping protocol
+TcpSlaveAgentListener.PingAgentProtocol.displayName=Protocolo de ping
+# Plugins Failed To Load
+PluginWrapper.PluginWrapperAdministrativeMonitor.DisplayName=Falha ao carregar o plugin
+# Malformed Test URL {0}.
+ProxyConfiguration.MalformedTestUrl=URL de test mal formatada {0}.
+# \u2018~\u2019 is only supported in a Unix shell and nowhere else.
+FilePath.TildaDoesntWork=\u2018~\u2019 \u00E9 apenas suportado em um shell Unix e em nenhum outro lugar.
+# Invalid Plugin Configuration
+PluginManager.PluginUpdateMonitor.DisplayName=Configura\u00E7\u00E3o do plugin inv\u00E1lida
+# {0} v{1} failed to load.
+PluginWrapper.failed_to_load_plugin={0} v{1} falhou ao carregar.
+# Cyclic Dependencies Detector
+PluginManager.PluginCycleDependenciesMonitor.DisplayName=Detector c\u00EDclico de depend\u00EAncias
+# {0} v{1} is disabled and older than required. To fix, install v{2} or later and enable it.
+PluginWrapper.disabledAndObsolete={0} v{1} est\u00E1 desastivado e mais antigo do que o requerido. Para corrigir, instale v{2} ou superior e habilite ele.
+# {0} v{1} is older than required. To fix, install v{2} or later.
+PluginWrapper.obsolete={0} v{1} \u00E9 mais antigo do que o requerido. Para corrigir, instale v{2} ou superior.
+# \u2018{0}\u2019 doesn\u2019t match anything because it is treated case sensitively. You can deactivate case sensitivity to get matches
+FilePath.validateAntFileMask.matchWithCaseInsensitive=
+# You must update Jenkins from v{0} to v{1} or later to run this plugin.
+PluginWrapper.obsoleteCore=Voc\u00EA deve atualizar o Jenkins de v{0} para v{1} ou superior para executar este plugin.
+# \u2018{0}\u2019 doesn\u2019t match anything: even \u2018{1}\u2019 doesn\u2019t exist
+FilePath.validateAntFileMask.doesntMatchAnythingAndSuggest=\u2018{0}\u2019 n\u00E3o combina nada: o   \u2018{1}\u2019 n\u00E3o existe
+# {0} v{1} failed to load. Fix this plugin first.
+PluginWrapper.failed_to_load_dependency={0} v{1} falhou ao carregar. Primeiro corrija este plugin.
```

Comment:
"Primeiro corrija este plugin." would be better re-worded as "Corrija o problema com este plugin primeiro" IMO.

Reply:
done